src/Celula.cpp

```diff
@@ -10,34 +10,34 @@ const bool VIVA = true;
 
 #include "Celula.h"
 
-
-Celula::Celula(){
+Celula::Celula() {
 	this->estadoCelula = MUERTA;
 	this->energiaCelula = 0;
 	this->RGBCelula = RGB();
```

Comment:
Tengo mi duda acá, si en el .h ya definis que RGBCelula es del tipo instancia de la clase RGB ya no lo inicializa ? no estoy seguro pero podes probar?
osea sacar esta linea y probar que funciona this->RGBCelula directamente en en los otros lados donde lo usas

Reply:
Vos decis que por tener un solo metodo constructor que no recibe parametro alguno, al declararla ya se inicializa?
Por que el this->RGBCelula a mi manera de verlo siempre va a funcionar porque solo estoy refiriendo a un atributo de clase Celula

Reply:
Estuve probando y creo no equivocarme al decir que tener razon. Ya al declararla se construye sola, no hace falte llamar a RGB();